src/Microsoft.TestPlatform.CoreUtilities/PublicAPI/net7.0/PublicAPI.Shipped.txt

```diff
@@ -0,0 +1,13 @@
+﻿#nullable enable
```

Comment:
Is this file identical to the one for net6.0? Can we just rename the other file and include the same file for both tfms in the csproj? I don't think these files will grow apart in the future.

Reply:
https://github.com/microsoft/vstest/blob/main/src/Microsoft.TestPlatform.CoreUtilities/PublicAPI/net6.0/PublicAPI.Shipped.txt They seem identical on first look.